.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: Docfx V3 Release
+on:
+  push:
+    branches: [ v3-release ]
+jobs:
+  build:
+    name: Release
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - uses: actions/setup-dotnet@v1
+    - name: Build
+      shell: pwsh
+      run: ./build.ps1
+      id: build-binaries
+    - name: Test
+      run: dotnet test -c Release
+    - name: Create GtiHub release
+      uses: marvinpinto/action-automatic-releases@latest
```

Comment:
Update `README.md` _Binary Builds` section to link to the preview releases page.

Reply:
added